frontend/pages/policies/ManagePoliciesPage/components/PoliciesTable/PoliciesTableConfig.tsx

```diff
@@ -128,6 +136,12 @@ const generateTableHeaders = (options: {
                     backgroundColor="#3e4771"
                   >
                     This policy has been marked as critical.
+                    {isSandboxMode && (
```

Comment:
is this check correct? is it meant to render if `isPremiumTier` is true instead?